packages/paste-website/src/pages/articles/README

```diff
@@ -21,7 +23,7 @@ Each file must start with a Frontmatter section that controls the meta data of t
 
 - **Title** - Title of the article or post
 - **date** - `YYYY-MM-DD` Date at which it was published
-- **slug** - URL slug that the full article appears at, absolute to the root of the website domain
+- **slug** - URL slug that the full article appears at, absolute to the root of the website domain (must end with a `/`)
```

Comment:
curious to learn: why should the URL slug end with `/`?

Reply:
I'm not sure, but my page wasn't loading before I remembered to add it 😅 

@SiTaggart why does the URL always end in a `/`?

Reply:
🤷🏼 Gatsby thing. Been bitten by it many a time